Editor/API/Attributes/AvatarPlatform.cs

```diff
@@ -0,0 +1,74 @@
+﻿using UnityEngine;
+
+namespace nadena.dev.ndmf
+{
+    /// <summary>
+    /// Declares which platforms a plugin or pass will execute for. Plugins and passes which execute for "Generic" will
+    /// always execute; otherwise, you can declare a specific platform or platforms to support.
+    /// </summary>
+    public sealed class AvatarPlatform
+    {
+        public static AvatarPlatform Generic = AvatarPlatform.Named("Generic");
+        public static AvatarPlatform VRChat = AvatarPlatform.Named("VRChat");
+        public static AvatarPlatform UniVRM = AvatarPlatform.Named("UniVRM");
```

Comment:
Does "UniVRM" platform support both VRM0 and VRM1? (I think it's reasonable because plugins can detect whether a UniVRM avatar is VRM0 or VRM1)

In contrast, I agree this must be "UniVRM" because it allows existence of alternate VRM exporter implementations.

Reply:
Yes, the intent is to better support cases where multiple avatar platforms are in a single project; for compile time compatibility the Unity version define system should be used.